### PR TITLE
Subscription themed

### PR DESCRIPTION
**Pull request by tenisakb**

### CODE REVIEW

**Comment by EdmundMai** [comment]
Will the next step be to re-use this theme across multiple components?

File: src/components/subscription/subscription-slide.tsx [hunk 1]
```diff
@@ -1,8 +1,21 @@
 import React from "react";
-import styled from "styled-components";
+import styled, { ThemeProvider } from "styled-components";
 
 import ScreenContainer from "../shared/screen-container";
 
+const theme = {
```
Comment (CKH4) [inline src/components/subscription/subscription-slide.tsx]: Theme should be generic and should be somewhere else (maybe config?)

File: src/components/subscription/subscription-slide.tsx [hunk 2]
```diff
@@ -107,65 +119,67 @@ const Space = styled.div`
...
-          needs.
-        </Content>
-        <Space />
-        <ButtonWrapper>
-          <Button>CONTACT US</Button>
-        </ButtonWrapper>
-      </FlexCol>
-    </FlexGrid>
-  </ScreenContainer>
+  <ThemeProvider theme={theme}>
```
Comment (CKH4) [inline src/components/subscription/subscription-slide.tsx]: The provider should be many levels up, near the top of the application tree

File: src/components/subscription/subscription-slide.tsx (hunk 1 above)
Comment (CKH4) [reply src/components/subscription/subscription-slide.tsx]: I totally lied about this

Comment (tenisakb) [reply src/components/subscription/subscription-slide.tsx]: > Will the next step be to re-use this theme across multiple components?

@EdmundMai I am working on it.. I will re-use 

> Theme should be generic and should be somewhere else (maybe config?)

@CKH4 Ok, understand, I will move it

**Pull request merged**